lib/spack/spack/test/conftest.py

```diff
@@ -272,6 +272,17 @@ def clean_test_environment():
     ev.deactivate()
 
 
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
This should have a comment block like the others noting what/why it is needed.

Reply:
Oh I don't know if we actually want this in, it was just an experiment to see what happens to the CI if I enabled it, I think we need more internal buy-in before thinking about seriously merging this.
Should be a draft but I don't know if CI runs on those.

Reply:
For sure gitlab CI does not, I can't remember if unit tests do or not.

Reply:
Seems like GHA will run on a draft, so I'm going to make this a draft for now. thanks Ryan